.eslintrc

```diff
@@ -19,7 +20,6 @@
     "react/react-in-jsx-scope": "off",
     "react/prop-types": "off",
     "@typescript-eslint/no-explicit-any": "off",
-    "@typescript-eslint/no-floating-promises": "error",
```

Comment:
This is already enabled as part of `plugin:@typescript-eslint/recommended-requiring-type-checking`.